src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -3622,8 +3622,8 @@ public static String substringBetween(final String str, final String open, final
      * <pre>
      * StringUtils.splitPreserveAllTokens(null, *, *)            = null
      * StringUtils.splitPreserveAllTokens("", *, *)              = []
-     * StringUtils.splitPreserveAllTokens("ab de fg", null, 0)   = ["ab", "cd", "ef"]
-     * StringUtils.splitPreserveAllTokens("ab   de fg", null, 0) = ["ab", "cd", "ef"]
+     * StringUtils.splitPreserveAllTokens("ab de fg", null, 0)   = ["ab", "de", "fg"]
+     * StringUtils.splitPreserveAllTokens("ab   de fg", null, 0) = ["ab", ,"", "de", "fg"]
```

Comment:
The first one matches what I got using the `master` branch. But the second example has a problem with the quotes? Supposed to be `["ab", "", "", "de", "fg"]` I think.

Reply:
Yep, should be that.